src/components/Registration.vue

```diff
@@ -35,9 +35,11 @@ export default {
   display: flex;
   background-image: url('../assets/onboarding_background.png');
   background-size: cover;
-  width: 100%;
   height: 100%;
   font-size: 16px;
+  margin-left: -300px;
+  position: relative;
+  z-index: 2;
```

Comment:
I agree with your observations though, so I'm going to add "todos" in the documents, so that we can consider refactoring later (it's outside the scope of this PR) :v:.